model/licensing.py

```diff
@@ -1246,14 +1246,20 @@ def __repr__(self):
                          'delivery_mechanism_id', 'resource_id'),
     )
 
+# The uniqueness constraint doesn't enforce uniqueness when one of the
+# fields is null, and one of these fields -- resource_id -- is
+# _usually_ null. So we also need a unique partial index to properly
+# enforce the constraint.
```

Comment:
In this case the UniqueConstraint already exists, so the redundant unique index is converted to a unique partial index.